R/msg.R

```diff
@@ -180,15 +184,19 @@ error_tribble_non_rectangular <- function(cols, cells) {
   )
 }
 
+error_name_length_required <- function() {
+  "`n` must be specified, when the `names` attribute is `NULL`"
+}
+
 error_frame_matrix_list <- function(pos) {
   bullets(
     "All values in `frame_matrix()` must be atomic:",
     pluralise_commas("Found list-valued element(s) at position(s) ", pos, ".")
   )
 }
 
-error_tidy_names_arg <- function() {
-  "The `.tidy_names` argument must be NULL, TRUE, FALSE, or a function."
+error_name_repair_arg <- function() {
+  "The `.name_repair` argument must be NULL, \"none\", \"valid\", \"tidy\", or a function."
```

Comment:
Do we explicitly support `"none_passive"`? What's a good way to keep error message and option list in sync?

Reply:
I view `"none_passive"` as an ugly temporary measure that lives as long as the `validate` argument is tolerated. But it's strictly an internal matter.

I don't have a plan re: keeping the error message and the possible values of `.name_repair` in sync 🤔

Reply:
Wait, `"none_passive"` **is** very important as it is our long-term default.

I think this is the question: is it defined implicitly as "the thing that happens when you don't specify `.repair_names`" (remains an internal matter) or do we give it name the public can see?

Reply:
Maybe rename `"none_passive"` to `"check"` and expose?

Reply:
The supported values could be an argument to the error reporting function (to keep the definitions in one place).